src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const kindOfAction of actions) {
+    switch (kindOfAction.type) {
+      case 'addProperties':
+        Object.assign(state, kindOfAction.extraData);
+        break;
+      case 'removeProperties':
+        kindOfAction.keysToRemove.forEach(el => {
```

Comment:
el?

Reply:
el is element of array
Why not? As for me, this is just common and understandable practice for naming 
fixed 👌

Reply:
@YeMyr it's good idea when we don't have enough time to think about variable names, but in this case `key` would be a better option